### ISSUE TITLE
Impossible to specify destination specific option

### ISSUE THREAD

**Issue by etiennemtl** [segmentio/Analytics.NET#77]
Hi,

The client library is very well made, but it's currently impossible to add destinations specific option.
In my particular case, I want to be able to add or remove a Tag when using the Identify call and [Vero destination](https://segment.com/docs/destinations/vero/#tags). In the documentation it says to pass an object called `tags` in the Vero integration object for example: 

```
analytics.identify('324LKJF', {
    email: 'friends@segment.com'
}, {
    integrations: {
        Vero: {
            tags: {
                id: '235FAG', // This is optional! If not defined, we default to the event's userId (ie: 324LKJF)
                action: 'add',
                values: ['warriors', 'giants', 'niners']
            }
        }
    }
})
```

But the .NET library only support setting an individual integration to `true` or `false` via the `SetIntegration("Vero", true)` method.

It would be nice to not have to call the HTTP API endpoint in order to succesfully pass the tags.

Thanks

**Comment by f2prateek** [comment]
hey @etiennemtl you're right. This is an oversight, we'll fix this. I don't have an ETA, but please keep an eye on this issue for updates. We're tracking this internally as well (https://segment.atlassian.net/browse/LIB-212).

**Comment by f2prateek** [comment]
Closed by #80
